### PR TITLE
Wraps env['KA_LITE'] with a string

### PR DESCRIPTION
**Pull request by jtamiace**

### CODE REVIEW

**Comment by aronasorman** [comment]
To further explain this issue:

On windows, somehow `KALITE_DIR` gets turned into a unicode string instead of a normal string somewhere. `subprocess` doesn't like unicode strings in its env variables, and errors out.

File: bin/kalite [hunk 1]
```diff
@@ -71,7 +71,7 @@ kalite_path = os.path.abspath(kalite_path)
 
 env = {}
 env.update(os.environ.copy())
-env['KALITE_DIR'] = kalite_path
+env['KALITE_DIR'] = str(kalite_path)
```
Comment (benjaoming) [inline bin/kalite]: thanks! could you add a comment, what @aronasorman explained?

**Comment by aronasorman** [comment]
@jtamiace I just noticed that you have other commits in here. We can just make this the PR for what you're doing, or we can cherry pick those out.

**Comment by aronasorman** [comment]
NVM, "Files Changed" says it's only two changes anyway.

**Comment by benjaoming** [comment]
yeah, i don't get it... 6 commits and only 2 small changes... is something misbehaving?

**Comment by aronasorman** [comment]
@benjaoming develop might've been rebased with these changes before, thus changing the commit hash but keeping the set of files changed.

**Comment by aronasorman** [comment]
Oh, why was this closed @jtamiace?

**Comment by jtamiace** [comment]
oops, i had created two different branches for this issue and intended to delete the one i wasn't working off of....... but deleted this one accidentally

**Comment by benjaoming** [comment]
:+1: for the comment

**Pull request merged**